.github/tools

```diff
@@ -13,4 +13,4 @@ jsonnetfmt v0.17.0
 jsonnet-lint v0.17.0
 jb v0.4.0
 gojsontoyaml 0.0.1
-shellcheck 0.9.0
+shellcheck 0.10.0
```

Comment:
Do we need to update shellcheck for the marker to work? Not that it would do any harm but just curious about it.

Reply:
No that is unrelated. However we install shellcheck@latest and they updated recently. So tool install fails with us still referencing 0.9.0.
Fwiw I'll push the same change to main.

Reply:
@danielmellado anything holding this up?

Reply:
/lgtm

Reply:
all good, thanks for the explanation!